src/dplasmaaux.h

```diff
@@ -109,8 +109,7 @@ extern void *dplasma_pcomm;
 #define dplasma_error(__func, __msg) do { fprintf(stderr, "%s: %s\n", (__func), (__msg)); } while(0)
 #endif /* defined(DPLASMA_DEBUG) */
 
-#if defined(DPLASMA_HAVE_CUDA)
 #include "dplasmaaux_cuda.h"
```

Comment:
Why are these headers not protected ?

Reply:
protection is self contained in the header itself

Reply:
We had proper protection in place avoiding to include unnecessary files. What justify the need to made this change ?